samples/Basic_Serial/app/application.cpp

```diff
@@ -239,6 +240,29 @@ void handleCommand(const String& command)
 	}
 }
 
+void listPartitions()
```

Comment:
Partitioning should have its own sample but I guess you have already planned this and the code in Basic_Serial is only for initial tests.

Reply:
Yes, just being lazy. It was the first sample to hand which used SPIFFS so just used it for basic testing. I'll probably add something to hostTests for this once.